src/ansys/grantami/bomanalytics/bom_types/_bom_reader.py

```diff
@@ -24,16 +24,19 @@
 from types import ModuleType
 from typing import Any, Dict, Generic, Iterable, Optional, Type, TypeVar
 
+from xmlschema import XMLSchema
+
 from ._base_types import BaseType, HasNamespace
 
 TBom = TypeVar("TBom", bound=BaseType)
 TAny = TypeVar("TAny", bound=BaseType)
 
 
-class GenericBoMReader(Generic[TBom]):
+class _GenericBoMReader(Generic[TBom]):
```

Comment:
Renaming these to be private methods means that they can be imported into `__init__.py`, which makes them easier to import elsewhere.